### ISSUE TITLE
No support for multi-users

### ISSUE THREAD

**Issue by PookaMustard** [crystalct/RetroArch_PSX_CE#1]
Just started using the RetroArch CE this past week. So far, so good. Except for one thing. For some reason, RetroArch saves all of its filedata into its own USRDIR, which means all of the PS3's three users have access to the same data. Now, that isn't to say saving to USRDIR is a bad idea in some scenarios (it's great for sharing playlists across users), but for stuff like save files, save states and controller remaps, it might be a better idea for them to be per-user.

**Comment by Zoukambada** [comment]
I think you are right but I must add that also playlists and thumbnails would be interesting to be saved in Game Data Utility, which is a section dedicated to DLCs, UPDATES in commercial games. Webman MOD already uses Game Data Utility (in XMB) to install / update PS2CONFIGs.
Why do that? If for some reason you have to uninstall Retroarch or maybe update it with another PKG, you would loose every settings, art, thumbnail, hiscores, screenshot, etc... because it's attached to the Retroarch install folder.

**Comment by PookaMustard** [comment]
Yeah, it'd be much nicer that way actually. But maybe as Saved Data (PS3) not Game Data.
